app/src/common/css/variables/display.scss

```diff
@@ -0,0 +1,2 @@
+$Z-INDEX-DEFAULT: 1;
```

Comment:
I think it's better to name this file `z-index-variables.scss` as more descriptive

Reply:
done